src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -276,7 +276,7 @@ public void GetKeyValueOptionsList_UsingRealEmbeddedRulesJson()
             CheckHasOption("S1578.format=^[A-Za-z_-][A-Za-z0-9_-]+\\.(c|m|cpp|cc|cxx)$");
 
             options.Count().Should()
-                .BeGreaterOrEqualTo(39); // basic sanity check: v6.6 has 39 - not expecting options to be removed
+                .BeGreaterOrEqualTo(37); // basic sanity check: v6.13 has 37 - not expecting options to be removed
```

Comment:
> not expecting options to be removed

well, only unless it happens 😉 
rule `CommentRegularExpression` with `2` parameters was removed in `6.13` - https://jira.sonarsource.com/browse/CPP-2609